### PR TITLE
initial updates to fm-overlay readme

### PR DESCRIPTION
**Pull request by gfargo**
This pr should clearly illustrate the additional functionality within FM overlays plugin.   I happily welcome suggestions on how to make the language more clear or concise.
- adds priority section
- adds conditional section 
- extends styling section 
- adds overlay type details

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,26 +1,84 @@
...
+
+Rich Text overlays are created using the classic **WYSIWYG editor**.
+
+### Conditionals
+
+Overlay conditionals allow the user to target where on the site the overlay will display.  The following conditionals are currently supported:
+
+- is_home
+- is_front_page
+- *is_category*
```
Comment (jakewrfoster) [inline README.md:30]: why italics here?

File: README.md [hunk 2]
```diff
@@ -1,26 +1,84 @@
...
+- *has_category*
+- is_single
+- *is_page*
+- *is_tag*
+- *has_tag*
+
+Conditionals marked with *italics* allow the user to specifically target a page, category, or tag.
```
Comment (jakewrfoster) [inline README.md:37]: oh

File: README.md [hunk 3]
```diff
@@ -1,26 +1,84 @@
...
+
+### Priorities
+
+Each overlay has a display priority that can be set using the `menu_order` attribute.  The larger the `menu_order` the higher the display priority.  If no priorities have been set the *published date* is used as a fallback.
+
+
+
+### Specificity > Priority
+
+Additional specificity within display conditionals will always override a priority.
```
Comment (jakewrfoster) [inline README.md]: "Conditionals that target specific posts and terms operate on a higher priority than those that target generally."

File: README.md [hunk 4]
```diff
@@ -1,26 +1,84 @@
...
+
+
+
+### Priorities
+
+Each overlay has a display priority that can be set using the `menu_order` attribute.  The larger the `menu_order` the higher the display priority.  If no priorities have been set the *published date* is used as a fallback.
+
+
+
+### Specificity > Priority
```
Comment (jakewrfoster) [inline README.md]: `### Specificity is prioritized over Menu Order`

File: README.md (hunk 1 above)
Comment (gfargo) [inline README.md:30]: I was debating the best way to indicate that those conditions support specific targeting.  Not sure if changing to:

_is_category_ : supports category targeting

would be more understandable?

**Comment by jakewrfoster** [comment]
🍯

**Pull request merged**